kvs-key-value-pairs/kvs-key-value-pairs.js

```diff
@@ -0,0 +1,27 @@
+import cf from 'cloudfront';
+​
+// (Optional) Declare the ID of the key value store that you have associated with this function
+
+const kvsId = "a1b2c3d4-5678-90ab-cdef-EXAMPLE11111";
```

Comment:
If we put in a comment about referencing/attaching KVS to your function then we can remove this line.